DocsCode/NavigationTutorial/NavigationTutorialViewController.swift

```diff
@@ -0,0 +1,128 @@
+// #-code-snippet: navigation dependencies-swift
+import Mapbox
+import MapboxCoreNavigation
```

Comment:
Is it necessary to import all four of these? At least `MapboxCoreNavigation` should be able to be implicitly imported.

Reply:
Looks likes it is required from testing it out 😐